### ISSUE TITLE
Django's funding status

### ISSUE THREAD

**Issue by pydanny** [jayfk/fundingoss.com#28]
The project has enough money to pay a full-time triage person for the project, Tim Graham.  I wouldn't say they are `well funded`, but I will say they are `funded`.

**Comment by nayafia** [comment]
agreed...think low-hanging fruit on #1 might be to change "self funded" to "somewhat funded" to at least have an option that reflects the reality between "well funded" and "no funding".

**Comment by jayfk** [comment]
I've changed the funding status to `Somewhat Funded` and the type of funding to `Foundation`, see http://fundingoss.com/39/
